crates/ln-dlc-node/src/tests/mod.rs

```diff
@@ -192,6 +204,40 @@ async fn fund_and_mine(address: Address, amount: Amount) -> Result<()> {
     Ok(())
 }
 
+/// Calculate the "minimum" acceptable value for the outbound liquidity
+/// of the channel creator.
+///
+/// The value calculated is not guaranteed to be the exact minimum,
+/// but it should be close enough.
+///
+/// This is useful when the channel creator wants to push as many
+/// coins as possible to their peer on channel creation.
+fn min_outbound_liquidity_channel_creator(peer: &Node, peer_balance: u64) -> u64 {
```

Comment:
:+1: very good.. I think we will need something like this as a validation function on the frontend as well. I tested to open a position with a very small amount and run into the reserve issue.

Reply:
That is true, we can apply a similar logic when opening any channel.

What channel were you thinking off specifically. I suppose the coordinator-app channel.

Reply:
I was referring to the dlc channel.